### DISCUSSION TITLE
Utilise host crowdsec install?

### DISCUSSION THREAD

**Discussion by shanelord01** [ZoeyVid/NPMplus#664] [Q&A] [answered]
I have crowdsec installed on my host - how would I setup npmplus crowdsec to interface with it?

Thanks.

**Comment by Zoey2936** [comment, marked as answer]
Instead of `docker exec` run only `crowdsec cscli bouncers add npmplus -o raw` and for the paths try `/etc/crowdsec/acquis.d` instead of `/opt/crowdsec/conf/acquis.d` and please add these two collections: `ZoeyVid/npmplus crowdsecurity/appsec-virtual-patching`
